keycloak/themes/sbomify/login/login.ftl

```diff
@@ -0,0 +1,144 @@
+<#import "template.ftl" as layout>
```

Comment:
This is a nice idea, but let's tighten this with just a login prompt instead of all this messaging.